src/box/session.cc

```diff
@@ -375,3 +377,71 @@ generic_session_sync(struct session *session)
 	(void) session;
 	return 0;
 }
+
+API_EXPORT int
+box_session_id()
+{
+	return current_session()->id;
+}
+
+API_EXPORT bool
+box_session_exists(uint64_t sid)
+{
+	return (session_find(sid) != NULL);
+}
+
+API_EXPORT const char *
+box_session_peer(uint64_t sid)
+{
+	struct session *session = session_find(sid);
+	if (session == NULL)
+		session = current_session();
+	if (session == NULL)
+		return NULL;
+
+	int fd = session_fd(session);
+	if (fd < 0) {
+		return NULL;
+	}
+
+	struct sockaddr_storage addr;
+	socklen_t addrlen = sizeof(addr);
+	if (sio_getpeername(fd, (struct sockaddr *)&addr, &addrlen) < 0)
+		return NULL;
+
+	return sio_strfaddr((struct sockaddr *)&addr, addrlen);
+}
+
+API_EXPORT int
+box_session_sync()
```

Comment:
For functions not having an argument we specify `void` in the arglist. It actually makes difference making the final executable file smaller.